src/QueryRenderer.ts

```diff
@@ -46,6 +47,7 @@ class QueryRenderChild extends MarkdownRenderChild {
     private readonly app: App;
     private readonly events: Events;
     private readonly source: string;
+    private readonly sourcePath: string;
```

Comment:
The source is the instructions, like `not done\nshort mode`.

I could rename sourcePath to filePath maybe?

Reply:
Thank you.

I've done the rename, and added some explanatory comments too.

It turns out that the reason I used the name I did was because the value passed in is `context.sourcePath`... But this change definitely makes the code in `QueryRenderChild` clearer.